From 127c584e896d93f11f20dc738cbd4e9db4299b65 Mon Sep 17 00:00:00 2001
From: Didier Raboud <odyx@debian.org>
Date: Tue, 9 Aug 2016 18:11:22 +0200
Subject: Force LC_ALL=C environment variable when grepping the output of
 lpstat.

 Permits testing in non-"C" environments.
Origin: vendor
Last-Update: 2016-07-11
Patch-Name: tests-make-lpstat-call-reproducible.patch
---
 test/run-stp-tests.sh | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/test/run-stp-tests.sh b/test/run-stp-tests.sh
index e0ece645f..c960a672d 100755
--- a/test/run-stp-tests.sh
+++ b/test/run-stp-tests.sh
@@ -727,7 +727,7 @@ fi
 IPP_PORT=$port; export IPP_PORT
 
 while true; do
-	running=`../systemv/lpstat -r 2>/dev/null`
+	running=`LC_ALL=C ../systemv/lpstat -r 2>/dev/null`
 	if test "x$running" = "xscheduler is running"; then
 		break
 	fi
@@ -840,13 +840,13 @@ kill -HUP $cupsd
 while true; do
 	sleep 10
 
-	running=`../systemv/lpstat -r 2>/dev/null`
+	running=`LC_ALL=C ../systemv/lpstat -r 2>/dev/null`
 	if test "x$running" = "xscheduler is running"; then
 		break
 	fi
 done
 
-description="`../systemv/lpstat -l -p Test1 | grep Description | sed -e '1,$s/^[^:]*: //g'`"
+description="`LC_ALL=C ../systemv/lpstat -l -p Test1 | grep Description | sed -e '1,$s/^[^:]*: //g'`"
 if test "x$description" != "xTest Printer 1"; then
 	echo "Failed, printer-info for Test1 is '$description', expected 'Test Printer 1'." >>$strfile
 	echo "FAIL (got '$description', expected 'Test Printer 1')"
